tracer/test/Datadog.Trace.TestHelpers/SpanMetadataRules.cs

```diff
@@ -355,7 +355,7 @@ public static Result IsServiceStackRedis(this MockSpan span) => Result.FromSpan(
                 .Matches(Name, "redis.command")
                 .Matches(Type, "redis"))
             .Tags(s => s
-                .IsPresent("redis.raw_command")
+                .IsPresent("meta.db.statement")
                 .IsPresent("out.host")
                 .IsPresent("out.port")
                 .Matches("component", "ServiceStackRedis")
```

Comment:
[OTEL](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#mongodb) expects `db.system`=`"redis"`. I guess we should do the same?

Reply:
While I'm looking at the attributes, it looks like we should have `db.redis.database_index` if its a non-default value. Since we don't emit something like this today, this can be a separate, small task that can be up-for-grabs. I've just now created a JIRA task to track adding this new tag

Reply:
added `db.system` in 1daca47e72e74da677c88d3fdf338f1d113c69e3. I'll need to change it for all dbs in another PR